rust/worker/src/blockstore/arrow/block/delta.rs

```diff
@@ -268,13 +285,23 @@ mod test {
             delta.add(prefix, key.as_str(), value.as_str());
         }
         let size = delta.get_size::<&str, &str>();
-        block_manager.commit::<&str, &str>(&delta);
+        let block = block_manager.commit::<&str, &str>(&delta);
+        let mut values_before_flush = vec![];
+        for i in 0..n {
+            let key = format!("key{}", i);
+            let read = block.get::<&str, &str>("prefix", &key);
+            values_before_flush.push(read.unwrap().to_string());
+        }
+        let blocks = vec![block.clone()];
+        block_manager.flush(&blocks).await.unwrap();
+
         let block = block_manager.get(&delta_id).await.unwrap();
-        assert_eq!(size, block.get_size());
+        // TODO: enable this assertion after the sizing is fixed
+        // assert_eq!(size, block.get_size());
```

Comment:
I think this is a bad merge, this was commented out when it should be left in

Reply:
Thank you!